publiq-ui/pub-input.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="input-container">
+    <pub-label v-if="label" id="form-input">{{ label }}</pub-label>
+    <b-form-input
+      id="form-input"
+      v-model="value"
+      :placeholder="placeholder"
+      @input="handleInput"
+    ></b-form-input>
+  </div>
+</template>
+
+<script>
+  import PubLabel from '@/publiq-ui/pub-label';
+
+  export default {
+    components: {
+      PubLabel,
+    },
+    props: {
+      label: {
+        type: String,
+        default: '',
+      },
+      placeholder: {
+        type: String,
+        default: '',
+      },
+    },
+    methods: {
+      handleInput() {
+        this.$emit('input', this.value);
```

Comment:
Nit: Maybe `change` would be clearer? (Like we changed the `input` on checkbox to `toggle`)

Reply:
Isn't input logical for pub-input? In case of the searchbox, where it's being used it would be `handleChangeSearch` instead of `handleInputSearch`

Reply:
Fair enough 👍